__tests__/app.test.js

```diff
@@ -104,3 +104,23 @@ describe('/api/articles/99(404) & /api/articles/notAnId(400)', () => {
 		});
 	});
 });
+
+describe('/api/users', () => {
+	describe('GET', () => {
+		test('Status 200: returns an array of users', () => {
+			return request(app).get('/api/users').expect(200).then((response) => {
+				expect(response.body.users).toEqual(expect.any(Array));
+			});
+		});
+		test('Status 200: returns an array with the correct properties', () => {
```

Comment:
consider revising the wording of this test, as it's not the array's properties you're checking here

Reply:
Have reworded the test